client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInput.tsx

```diff
@@ -10,22 +10,17 @@ import AlphanumericTextField from 'commons/AlphanumericTextField/AlphanumericTex
 import useStyles from './AirportInputStyles';
 
 const AirportInput = (props: any) => {
-    const {
-        errors,
-        setError,
-        clearErrors,
-        country,
-        countryFieldName,
-        city,
-        cityFieldName,
-        airport,
-        airportFieldName,
-        handleChangeExposureDataAndFlightsField,
+    const { errors, setError, clearErrors, country, countryFieldName, city, cityFieldName, airport,
+            airportFieldName, handleChangeExposureDataAndFlightsField,
     } = props;
 
     const classes = useStyles();
+
+    const [countryToShow, setCountryToShow] = useState<Country | null>(country);
```

Comment:
why do you need this state, I think all the data in this specific tab is on the context

Reply:
I want to initialize the field with a default country, I set the state with the default country/country from DB, and then I can change the field to null without the country returning to the default country.